src/controllers/dynakube/activegate/capability/capability.go

```diff
@@ -5,11 +5,11 @@ import (
 	"strings"
 
 	dynatracev1beta1 "github.com/Dynatrace/dynatrace-operator/src/api/v1beta1"
+	"github.com/Dynatrace/dynatrace-operator/src/controllers/dynakube/activegate/statefulset"
 	corev1 "k8s.io/api/core/v1"
 )
 
 const (
-	MultiActiveGateName       = "activegate"
```

Comment:
was this removed because of circular dependency ?

Reply:
yes